README.md

```diff
@@ -12,4 +16,4 @@ APP_ID=<appId>
 APP_SECRET=<appSecret>
 ```
 - Start server: inside `server` directory run `npm start` (remember to `npm install` first)
-- Start client: inside `client` directory run `npm start` (remember to `npm install` first)
+- Start client: inside `client` directory run `npm run dev` (remember to `npm install` first)
```

Comment:
could we add a start script instead, similar to topdown-shooter?

Reply:
Sure thing, I'll do that now :)